internal/driver/checkstatuses.go

```diff
@@ -82,27 +82,35 @@ func (d *Driver) checkStatusOfDevice(device models.Device) {
 // Higher degrees of connection are tested first, because if they
 // succeed, the lower levels of connection will too
 func (d *Driver) testConnectionMethods(device models.Device) (status string) {
-
-	// sends get capabilities command to device (does not require credentials)
-	devClient, edgexErr := d.newTemporaryOnvifClient(device)
-	if edgexErr != nil {
-		d.lc.Debugf("Connection to %s failed when creating client: %s", device.Name, edgexErr.Message())
-		// onvif connection failed, so lets probe it
+	devClient, err := d.getOnvifClient(device)
```

Comment:
Is the previous comment for this line really not needed or was accidentally removed?

Reply:
in the past, the code created a new onvif client for every time check status was called. this was wasteful. now, if a client already exists for a camera, this client is used. Because of this, the get capabilities call is no longer made.